docs/design.md

```diff
@@ -0,0 +1,102 @@
+V2 Design Note
+==============
+
+- [Overview](#overview)
+  - [Background](#background)
+  - [Goals](#goals)
+- [NetworkNamespace Resource](#networknamespace-resource)
+- [Running Placemat2 inside a kubernetes Pod](#running-placemat2-inside-a-kubernetes-pod)
+- [New Features](#new-features)
+  - [New Package Placemat2](#new-package-placemat2)
+  - [Redfish API Support](#redfish-api-support)
+- [Obsolete Features](#obsolete-features)
+
+## Overview
+
+Placemat is a data center network and server simulation tool using QEMU/KVM virtual machines. This document elaborates the motivation and goals of Placemat v2.
+
+### Background
+
+Currently, the integration test suites of Neco, our data center management software, are running on a virtual data center built with Placemat v1 on a GCP instance, and they are unstable due to problems caused by nested VMs.
+To work around the issue, we're considering running the integration test suits in our Kubernetes cluster build on bare-metal servers instead of a GCP instance.
```

Comment:
```suggestion
To work around the issue, we're considering running the integration test suits in our Kubernetes cluster built on bare-metal servers instead of a GCP instance.
```